reddit_gold/templates/snoovatar.html

```diff
@@ -64,6 +64,14 @@
       <div class="username">${thing.username}</div>
     %endif
     <p>${_md('view and discuss snoovatars in /r/snoovatars')}</p>
+    %if c.user.gold and c.user.name != thing.username:
+      ${_md('create your own snoovatar [here](/user/%(username)s/snoo)' % dict(username=c.user.name))}
+    %elif not c.user.gold:
+      <div class='snoovatar-footer'>
```

Comment:
Nit: doublequotes for attributes are preferred - https://github.com/reddit/styleguide/blob/master/html/README.md#style